ChildProject/projects.py

```diff
@@ -16,6 +16,17 @@
 )
 from .utils import get_audio_duration, path_is_parent
 
+RAW_RECORDINGS = os.path.normpath("recordings/raw")
+CONVERTED_RECORDINGS = os.path.normpath("recordings/converted")
+STANDARD_SAMPLE_RATE = 16000
+STANDARD_PROFILE = 'standard' # profile that is expected to contain the standardized audios (16kHz). The existence and sampling rates of this profile are checked when <validating this profile> or <validating without profile and the raw recordings are not 16kHz>.
+
+METADATA_FOLDER = 'metadata'
+CHILDREN_CSV = 'children.csv'
+RECORDINGS_CSV = 'recordings.csv'
+
+PROJECT_FOLDERS = ["recordings", "annotations", "metadata", "doc", "scripts"]
```

Comment:
I'm noticing all of these variables are in all caps. Is that what is done elsewhere in the package? does it have a specific meaning? is it better this way or a different way?

Reply:
ps great idea to use normpath - is that what we do elsewhere?

Reply:
All caps is reserved for fixed variables, i a way to indicate that you can change easily entire aspects of the package by changing those variables without breaking anything. My change here:
- for those existing, take them out of the class and put them in all caps as they should not be built each time we create a project object, it is better practice to have them as constants at the level of the module (file)
- for those not existing, replace hard coded strings in the code by them to make changes easier. This way if one day we change 'children.csv' with 'particpants.csv' , it should just be a matter of changing that constant and everything should work (probably not yet as there is a lot of harmonization to be done on the whole package)

Reply:
I started to use normpath in an effort to have automated windows testing (which we do have and maintain now). There are some areas of the package where normpath is not really integrated correctly to support all of that but they work because libraries are smart enough to use linux paths in windows. But it would definitely be better to make that the norm

Reply:
sounds great - one more thing to add to the "todo" list for may-july!